packages/babel-plugin-remove-graphql-queries/src/__tests__/index.js

```diff
@@ -9,6 +9,30 @@ function matchesSnapshot(query) {
   expect(code).toMatchSnapshot()
 }
 
+it(`Works correctly with the kitchen sink`, () => {
+  matchesSnapshot(`
+  import React from 'react'
+  import { graphql, useStaticQuery, StaticQuery } from 'gatsby'
+
+  export default () => {
+    const query = graphql\`{site { siteMetadata { title }}}\`
+    const siteDescription = useStaticQuery(query)
+
+    return (
+      <StaticQuery
+        query={graphql\`{site { siteMetadata { title }}}\`}
```

Comment:
Maybe make this query different? If the content of the query is the hash, it's probably a good idea that we ensure the hash changes.

Reply:
Oh, yeah. Missed that.

Reply:
Also want to be clear here that we can isolate this PRs changes to _just_ useStaticQuery. I wouldn't necessarily say you have to make a ton of other improvements _unless_ they're necessary for the implementation.

I do like the kitchen sink test though (even if technically "broken"). One thing we can do is use `test.todo` introduced in Jest 24, and mark it as a todo for future implementation.

Reply:
`it.todo` is awesome! Done!